pipelines/src/pipelines/__init__.py

```diff
@@ -30,4 +30,10 @@ def generate_query(input_file: Path, **replacements) -> str:
     with open(input_file, "r") as f:
         query_template = f.read()
 
-    return Template(query_template).render(**replacements)
+    # Render the template with the provided replacements
+    query = Template(query_template).render(**replacements)
+
+    # Escape double quotes, newline and tab characters
+    query = query.replace('"', '\\"').replace("\n", "\\n").replace("\t", "\\t")
```

Comment:
is the first one correct? there are two `\`

Reply:
Well, the reason behind this new line is:

- In the old component, the query that we needed to pass was a simple string, where we already were generated \n and \t characters.

- In the new component, this string live inside a JSON, and for loading the query executes a `json.loads` command that fails in interpreting the escaping commands for some reason I don't fully understand. So, if we want to preserve the exact format of the string, we need to escape also the special characters.

Reply:
Is it just the double-quotes issue? What happens if we only use single quotes `'` in the SQL script?

Reply:
mmm it seems it worked :) `json.loads` was loading properly the line breaks but not the quotes. Sorry I forgot to check that!